src/shared.js

```diff
@@ -1583,6 +1585,38 @@ export const updateCollectionSettingData = async (biospecimenData, tubes, partic
             }
         });
     }
+    else if (settings[visit][conceptIds.baseline.bloodCollectedTime] !== '' ||  settings[visit][conceptIds.clinicalDashboard.bloodCollectedTime] !== ''){
+        const participantBloodCollected = participantData[conceptIds.baseline.bloodCollected] === conceptIds.yes;
+        const collectionSetting = biospecimenData[conceptIds.collection.collectionSetting];
+        const isResearch = collectionSetting === conceptIds.research;
+        const isClinical = collectionSetting === conceptIds.clinical;
+        let totalBloodTubesAvail = 6; // 6 blood tubes are available. Update this number if more blood tubes are added
```

Comment:
I would rather not have this hardcoded, don't want to have to remember to update this if / when more tubes are added.